command/job_init.go

```diff
@@ -36,6 +37,12 @@ Init Options:
 
   -connect
     If the connect flag is set, the jobspec includes Consul Connect integration.
+
+  -template
+    Specifies a predefined template to initialize. Must be a Nomad Variable that lives at nomad/job-templates/<template>
+
+  -list-templates
+    Display a list of possible job templates to pass to -template. Reads from all variables pathed at nomad/job-templates/<template>
```

Comment:
Unfortunately we don't have automatic wrapping to the terminal for these, so let's hard-wrap them to something reasonable like 80 char (or aligned with the rest of the help block, at least)